pyorientAssignment.py

```diff
@@ -36,14 +37,29 @@
 # then test each price to see if it is smaller than the current minimum. If it is, update 
 # the minimum variable with that price. You can do something similar to find the maximum.
 
+min_p = 900
+max_p = 0
+sum_p = 400
 for record in records:
-	print record.price
-
+        print record.price
+        #sum_p += record.price
+	if record.price < min_p:
+	    min_p = record.price
+	    #ini_min = min_price
+	else:
```

Comment:
The else statements here and on line 55 are redundant. It won't mess up the code but they are not necessary.